net-misc/eventd/eventd-0.21.0.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit linux-info systemd
+
+DESCRIPTION="A small daemon to act on remote or local events"
+HOMEPAGE="https://www.eventd.org/"
+SRC_URI="https://www.eventd.org/download/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3+ LGPL-3+ MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug fbcon +introspection ipv6 libcanberra libnotify +notification
+	purple sound speech ssdp systemd test websocket +X zeroconf"
+
+REQUIRED_USE="
+	X? ( notification )
+	fbcon? ( notification )
+	notification? ( || ( X fbcon ) )
+"
+
+COMMON_DEPEND="
+	dev-libs/glib:2
```

Comment:
You don't know on which machine this package is going to be installed. It might or might not be fully updated and the package not being in the tree anymore is no guarantee that it was actually updated everywhere. It all depends on package manager options to pull packages upgrades or not if a new dependency specifically require a new enough version. A package might stay at a lower version forever if no installed packages specifically requires that a new one be installed, hence it is good practice to always have a minimal version if one is provided.